docs/subgraph/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Aragon Subgraph [WIP]
+sidebar_label: Introduction
+sidebar_position: 0
+---
+
+## The On-Chain Information
+
+In Aragon we use [The Graph](https://thegraph.com/docs/en/about/), a decentralized protocol for indexing and querying blockchain data. The Graph makes it possible to query data that is difficult to retrieve directly from the Smart Contracts.
+
+The Aragon Subgraph stores all DAO-related information, either to help users to manage and interact with their DAOs or to query useful information, such as its treasury information, members, plugins, etc.
+
+On this site, can be found help with [queries](./01-queryExample/index.md) as well as all the documentation for our [subgraph schema](./reference-guide/index.md)
+
+## Playground
+
+We encourage users to test the [Aragon subgraph-satsuma playground](https://subgraph.satsuma-prod.com/aragon/osx-mainnet/playground) to fully understand what information is available and how to access it.
```

Comment:
```suggestion
You can use the [Aragon subgraph-satsuma playground](https://subgraph.satsuma-prod.com/aragon/osx-mainnet/playground) for testing.
```

Sometimes speaking directly to the user (using "you") instead of using the more distanced "we" is clearer and sounds less distanced.